FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -42,7 +34,24 @@ export function SignUpFormContentComponent(props) {
   });
 
   const { setIsValid } = props;
+  // modal start
+  const RulesModal = React.lazy(() => import('./RulesModal'));
```

Comment:
It's better to do lazy import outside of the component since you might hit it on every rerender

Reply:
Still not fixed. Every time `SignUpFormContentComponent` get rerendered you will run `const RulesModal = React.lazy(() => import('./RulesModal'));`

Example - https://codesandbox.io/s/nvsn7g?file=/src/App.js&utm_medium=sandpack

Reply:
Done  import RulesModal from './RulesModal';